### PR TITLE
feat(ontology): display description for default and existing props (DSP-1154)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1154

### CODE REVIEW

**Comment by kilchenmann** [comment]
The description from the default properties will be displayed as tooltipp:

Screen Shot 2021-06-29 at 14 58 42

@mdelez could you check the english sentences? Thanks

**Comment by gautschr** [comment]
I suggest for "Text":
"Short text such as" instead of "Small text like"

A question concerning Date/Time: The inclusion of the new GUI element which allows for a proper display of dates with lower precision than one day - is this another PR?

**Comment by kilchenmann** [comment]
> I suggest for "Text":
> "Short text such as" instead of "Small text like"

I will update as suggested.

> 
> A question concerning Date/Time: The inclusion of the new GUI element which allows for a proper display of dates with lower precision than one day - is this another PR?

The new gui element for dates should already be implemented. So yes, how should the description look like?

File: src/app/project/ontology/default-data/default-properties.ts [hunk 1]
```diff
@@ -192,6 +207,7 @@ export class DefaultProperties {
                 {
                     icon: 'language',
                     label: 'External URL',
+                    description: 'Link to an external http address',
```
Comment (mdelez) [inline src/app/project/ontology/default-data/default-properties.ts:210]: perhaps using the word "website" would be clearer than "http address"

File: src/app/project/ontology/default-data/default-properties.ts [hunk 2]
```diff
@@ -155,6 +168,7 @@ export class DefaultProperties {
                 {
                     icon: 'functions',
                     label: 'Decimal',
+                    description: 'Floating point number',
```
Comment (mdelez) [inline src/app/project/ontology/default-data/default-properties.ts:171]: I would simply say "Decimal value". I've only used the term "Floating point number" as a developer.

File: src/app/project/ontology/default-data/default-properties.ts [hunk 3]
```diff
@@ -68,39 +72,43 @@ export class DefaultProperties {
...
-                },
-                {
-                    icon: 'radio_button_checked',
-                    label: 'Multiple choice',
-                    subPropOf: Constants.HasValue,
-                    objectType: Constants.ListValue,
-                    guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'Radio',
-                    group: 'List'
                 }
+                // {
```
Comment (mdelez) [inline src/app/project/ontology/default-data/default-properties.ts:81]: is this code still needed?

**Review by mdelez — CHANGES_REQUESTED** [review]

File: src/app/project/ontology/default-data/default-properties.ts (hunk 3 above)
Comment (kilchenmann) [reply src/app/project/ontology/default-data/default-properties.ts:81]: I'm not sure if we have to reactivate it one day!?

File: src/app/project/ontology/default-data/default-properties.ts [hunk 4]
```diff
@@ -218,18 +235,11 @@ export class DefaultProperties {
                 {
                     icon: 'palette',
                     label: 'Color',
+                    description: 'A simple color value',
```
[the remark this answers is not in the document]
Comment (kilchenmann) [reply src/app/project/ontology/default-data/default-properties.ts:238]: I am not sure if everyone understands the term "hexadecimal". Apart from that, in the GUI we have the color picker only and the hexadecimal value itself is not displayed anymore (s. DSP-1667)

File: src/app/project/ontology/property-form/property-form.component.spec.ts [hunk 5]
```diff
@@ -61,6 +61,7 @@ class SimpleTextHostComponent {
         'propType': {
             'icon': 'short_text',
             'label': 'Short',
+            'description': 'Small text like title',
```
[the remark this answers is not in the document]
Comment (kilchenmann) [reply src/app/project/ontology/property-form/property-form.component.spec.ts:64]: already done in 6fb9746

[the remark this answers is not in the document]
Comment (kilchenmann) [reply src/app/project/ontology/property-form/property-form.component.spec.ts]: Done in 1ac8922

File: src/app/project/ontology/default-data/default-properties.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (mdelez) [reply src/app/project/ontology/default-data/default-properties.ts:238]: That's fair :)

**Review by mdelez — APPROVED** [review]

**Comment by gautschr** [comment]
We need a kind of dd/mm/yyyy BCE/CE icon and for the short description I suggest "dd/mm/yyyy format; for historical dates"

**Pull request merged**